zebrad/src/components/mempool/storage.rs

```diff
@@ -78,6 +81,12 @@ pub enum SameEffectsChainRejectionError {
     #[error("best chain tip has reached transaction expiry height")]
     Expired,
 
+    #[error("transaction outpoints or nullifiers were committed to the best chain")]
```

Comment:
Trivial clarification:

```suggestion
    #[error("transaction inputs were spent, or nullifiers were revealed, in the best chain")]
```